slicebase/init/setup_rsync.sh

```diff
@@ -2,6 +2,20 @@
 
 SLICENAME=$1
 
+# A list of Google Cloud netblocks. Generated from DNS-based SPF records.  To
+# regenerate this list from DNS, you can run the command:
+#   nslookup -q=TXT _cloud-netblocks.googleusercontent.com  8.8.8.8 \
+#    | grep text \
+#    | sed -e 's/.*=spf1 //' -e 's/?all.*//' -e 's/include://g' -e 's/ /\n/g'  \
+#    | while read; do nslookup -q=TXT $REPLY 8.8.8.8; done \
+#    | grep 'text = ' \
+#    | sed -e 's/.*spf1 //' -e 's/ ?all.*//' -e 's/ /\n/g' \
+#    | grep ^ip4 \
+#    | sed -e 's/ip4://' \
+#    | xargs \
+#    | sed -e 's/ /, /g'
```

Comment:
I had that thought as soon as I hit "comment". So, friendly push-back part dieux: What about putting the nslookup in a time-bounded if; if the if succeeds, update the static list; if the if fails, rely on the previously-derived static list?